pkg/doc/did/publickey.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package did
+
+// PublicKey must include id and type properties, and exactly one value property
+type PublicKey map[string]interface{}
```

Comment:
Is there a case for a non-string storage in PublicKey?

Are there mandatary fields that would be best represented as a struct?

Reply:
done